Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Use an official Node.js runtime as a parent image
+FROM node:18-alpine AS build
```

Comment:
Šeit kāds specifisks iemesls 18 versijai? It kā lokāli maucu ar 20

Reply:
Pašpikoju vecākus piemērus nepaskatoties aktuālu versiju. Pamainīju, patestēju un strādā ar node:20.